src/hooks/tradingView/useTradingView.ts

```diff
@@ -135,9 +142,17 @@ export const useTradingView = ({
               });
 
               orderbookCandlesToggleRef.current = tvWidgetRef.current.createButton();
-              orderbookCandlesToggleRef.current.innerHTML = `<span>${`${ohlcTitle}*`}</span> <div class="ohlcButton-toggle"></div>`;
+              orderbookCandlesToggleRef.current.innerHTML = `<span>${`${ohlcTitle}*`}</span> <div class="toggle"></div>`;
               orderbookCandlesToggleRef.current.setAttribute('title', ohlcBody as string);
             }
+            if (buySellMarksToggleRef) {
+              buySellMarksToggleRef.current = tvWidgetRef.current.createButton();
+              buySellMarksToggleRef.current.innerHTML = `<span>Buys/Sells</span> <div class="toggle"></div>`;
```

Comment:
can we localize `Buys/Sells`? i.e. like ORDER_LINES does [it](https://github.com/dydxprotocol/v4-web/pull/927/files#diff-c8b3ff077660b9f0fa861c122ef4a719a9d312fd37ac918cdadca357f568f285R128)

Reply:
OMG yes i forgot! i even added it to v4-localization already 😂